Makefile

```diff
@@ -27,13 +27,15 @@ GVDDK:= github.com/vmware-tanzu/astrolabe/vendor/github.com/vmware/gvddk
 VDDK_LIBS:= $(GVDDK)/vmware-vix-disklib-distrib/lib64
 
 # The binary to build (just the basename).
-PLUGIN_BIN ?= $(wildcard velero-*)
-DATAMGR_BIN ?= $(wildcard data-manager-*)
+PLUGIN_BIN ?= velero-plugin-for-vsphere
+DATAMGR_BIN ?= data-manager-for-plugin
+BACKUPDRIVER_BIN ?= backupdriver
```

Comment:
Sorry to be picky, but I think we want this as backup-driver to be consistent with the other apps

Reply:
The package is called backupdriver under velero-plugin-for-vsphere/cmd. That name is picked up by the build script in https://github.com/vmware-tanzu/velero-plugin-for-vsphere/blob/master/hack/build.sh#L79 using the line

backupdriver: astrolabe
	@echo "making: $@"
	$(MAKE) build BIN=$(BACKUPDRIVER_BIN) VERSION=$(VERSION)

So we need to keep "BACKUPDRIVER_BIN ?= backupdriver" to pick the right package. Should we change the package name to cmd/backup-driver?